IBPSA/ThermalZones/ReducedOrder/RC/OneElement.mo

```diff
@@ -386,7 +386,7 @@ equation
   connect(volMoiAir.heatPort, senTAir.port)
     annotation (Line(points={{-20,-16},{-20,0},{80,0}}, color={191,0,0}));
   connect(senTAir.T, TAir)
-    annotation (Line(points={{100,0},{108,0},{108,160},{250,160}},
+    annotation (Line(points={{101,0},{108,0},{108,160},{250,160}},
```

Comment:
I think this little changes shouldn't be merged, right?

Reply:
No, this actually belongs to another package. Not sure how it happened.